.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  check-links:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@main
+      - name: Markup Link Checker (mlc)
+        uses: becheran/mlc@v0.15.4
+        with:
+          args: --no-web-links
```

Comment:
Does this option prevent the checker from checking links going out to the WWW? If yes, why would we want this option to be enabled for this repo?

Reply:
The problem with not using this option might be that you get more false positives. For example if an external server is down for a few hours the link would be reported as broken. But you could also give it a try without this option and see if it is helpful or annoying 😉